sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -5354,14 +5354,102 @@ pi_result cuda_piextUSMEnqueueMemset2D(pi_queue, void *, size_t, int, size_t,
   return {};
 }
 
-// TODO: Implement this. Remember to return true for
-//       PI_EXT_ONEAPI_CONTEXT_INFO_USM_MEMCPY2D_SUPPORT when it is implemented.
```

Comment:
Remember this, otherwise the function won't be called. 😉

Reply:
https://github.com/intel/llvm/pull/7941/commits/e26ce84887bda00ca5cc8e6caafd062fd414a734